src/pages/api/login.ts

```diff
@@ -0,0 +1,31 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+type ResponseData = {
+  message: string;
+};
+
+const loginUser = (requestData: { email: string; password: string }) =>
+  fetch("https://sportujspolu-api.onrender.com/api/v1/user/login", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify(requestData),
+  });
+
+export default async function handler(
+  req: NextApiRequest,
```

Comment:
```suggestion
  {body: {email, password }: NextApiRequest,
```
You can keep what you have if you prefer it. No strong opinion. Just showing possibilities

Reply:
changed also in me.ts